BaragonAgentService/src/main/java/com/hubspot/baragon/agent/managers/AgentRequestManager.java

```diff
@@ -87,12 +77,14 @@ public Response processRequest(String requestId, Optional<RequestAction> maybeAc
         default:
           return apply(request, maybeOldService);
       }
+    } catch (LockTimeoutException e) {
+      LOG.warn(String.format("Could not acquire lock before timeout for request %s", requestId), e);
+      return Response.status(Response.Status.CONFLICT).build();
```

Comment:
could be useful if we had an explicit error message in here (i.e. `"Couldn't acquire agent lock for {requestId} in {timeout}"`)